src/2048-expert/2048-expert.css

```diff
@@ -98,16 +106,22 @@ h1 {
  }
 
  /* add colour to cells */
+ [data-digits="0"] {
+   color: rgb(175, 161, 132) !important
```

Comment:
Can you add a comment describing why the `!important` is needed / what it does?

Reply:
Sure